content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -572,6 +572,43 @@ ConfigMap updates.
 
 {{% capture discussion %}}
 
+## Automatically Created ConfigMaps
+
+{{< feature-state for_k8s_version="v1.13" state="stable" >}}
+
+If BoundServiceAccountTokenVolume was enabled, kube-controller-manager would create a ConfigMap named `kube-root-ca.crt` in each namespace.
+
+Verify that the `kube-root-ca.crt` ConfigMap exists in `default` Namespace:
+
+```shell
+kubectl get cm -n default
+```
+
+The output is similar to this:
+
+```shell 
```

Comment:
This shouldn't use `shell` as it's not code that `bash` etc could run.

Reply:
Yeah, but we need this to outstanding the output.